README.asciidoc

```diff
@@ -523,19 +523,11 @@ To properly use snippets, you'll probably want something like this:
 
 [source,kak]
 ----
-def -hidden insert-c-n %{
- try %{
-   lsp-snippets-select-next-placeholders
-   exec '<a-;>d'
- } catch %{
-   exec -with-hooks '<c-n>'
- }
-}
-map global insert <c-n> "<a-;>: insert-c-n<ret>"
+map global insert <c-a-n> '<a-;>: lsp-snippets-select-next-placeholders<ret>'
+map global normal <c-a-n> ': lsp-snippets-select-next-placeholders<ret>'
```

Comment:
I have never used it but the old c-n seems nice because of the simple UI,
Maybe we should map c-n (or tab?) by default eventually. Usually, plugins aren't supposed to add key bindings but kak-lsp ought to be an exception.

I think the old behavior still works if we just tweak the try-block a bit:

	define-command -override -hidden insert-c-n %{
		try %{
			lsp-snippets-select-next-placeholders
		} catch %{
			exec -with-hooks '<c-n>'
		}
	}

Reply:
Ok I realized that this c-n binding will break cyclying through completions, probably there's an easy fix

Reply:
That is why i removed it. This issue, and simply trying to simplify the example.
This is what i use, in my own `lsp-setup` function:

```kak

  hook -group lsp window InsertCompletionShow .* %{
    unmap window insert <c-n>
  }
  hook -group lsp window InsertCompletionHide .* %{
    map window insert <c-n> '<a-;>: lsp-snippets-select-next-placeholders<ret>'
  }
  map window insert <c-n> '<a-;>: lsp-snippets-select-next-placeholders<ret>'
  map window normal <c-n> ': lsp-snippets-select-next-placeholders<ret>'
```

Reply:
this sounds great, we should probably make it the default behavior